### PR TITLE
DOCS Update security release docs / satis instructions

### PR DESCRIPTION
**Pull request by tractorcow**
@dhensby I've attempted to reproduce the security release process you wrote for cow 1.0. Do you think the below is a fair representation of the general process?

### CODE REVIEW

File: docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md [hunk 1]
```diff
@@ -134,6 +139,44 @@ a public stable, not an RC or dev-branch. Security warnings that do not require
...
 
+### Setting up satis for hosting private security releases
+
+When installing a project from protected repositories, it's necessary prior to creating your project
+to override the public repository URLs with the private repositories containing undisclosed fixes. For
+this we use [satis](https://github.com/composer/satis).
+
+To setup a Satis project for a release:
+
+* Ensure Satis is installed globally: `composer global require composer/satis dev-master`
```
Comment (robbieaverill) [inline docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md]: Suggestion: use `1.0-dev` alias instead of dev-master

**Comment by tractorcow** [comment]
PR for cow so that it works properly https://github.com/silverstripe/cow/pull/62

This should make our next release easier @robbieaverill :)

File: docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md [hunk 2]
```diff
@@ -29,7 +29,10 @@ As a core contributor it is necessary to have installed the following set of too
   be installed in a global location via the below command. Please see the installation
   docs on the cow repo for more setup details.
   `composer global require silverstripe/cow dev-master`
-* [transifex client](http://docs.transifex.com/client/). 
+* [satis repository tool](https://github.com/composer/satis). This should be installed
+  globally for minimum maintenance.
```
[the remark this answers is not in the document]
Comment (tractorcow) [reply docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md]: No, satis is the tool (like composer), you create the actual archive separately to the satis install location.

File: docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md [hunk 3]
```diff
@@ -134,6 +141,59 @@ a public stable, not an RC or dev-branch. Security warnings that do not require
...
 
+### Setting up satis for hosting private security releases
+
+When installing a project from protected repositories, it's necessary prior to creating your project
+to override the public repository URLs with the private repositories containing undisclosed fixes. For
+this we use [satis](https://github.com/composer/satis).
+
+To setup a Satis project for a release:
+
+* Ensure Satis is installed globally: `composer global require composer/satis ^1@dev`
```
[the remark this answers is not in the document]
Comment (tractorcow) [reply docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md]: You'll need to install an older version (e.g. 1@alpha)

[the remark this answers is not in the document]
Comment (tractorcow) [reply docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md]: I've updated the install instructions.

File: docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dhensby) [reply docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md]: Hmm, ok - well, that's not what their official docs say; their installation instructions are to use `composer create-project` command, not a require command, I think that's especially true as they ship a `composer.lock` file which is ignored when installed via `composer require`

File: docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md [hunk 4]
```diff
@@ -29,7 +29,10 @@ As a core contributor it is necessary to have installed the following set of too
   be installed in a global location via the below command. Please see the installation
   docs on the cow repo for more setup details.
   `composer global require silverstripe/cow dev-master`
-* [transifex client](http://docs.transifex.com/client/). 
+* [satis repository tool](https://github.com/composer/satis). This should be installed
+  globally for minimum maintenance.
+  `composer global require composer/satis ^1@dev` (or `^1@alpha` for php 5)
```
Comment (dhensby) [inline docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md]: I don't actually think the `@alpha` works... at least alpha2 doesn't and I've opened an issue about it which was responded to with "use master not tags"

Also, 1.0.0 has now been tagged stable, so we probably shouldn't recommend `@dev`

**Comment by dhensby** [comment]
A couple of comments from me on this. I'd be tempted to actually run through some of these instructions to make sure they actually work and aren't assumptions.... (eg: that using an alpha is ok or that installing globally is acceptable).

**Comment by robbieaverill** [comment]
@dhensby keen for your thoughts on this after your last release

**Comment by robbieaverill** [comment]
@unclecheese might be cool to get your thoughts on this after your last round of releases too

**Comment by tractorcow** [comment]
@unclecheese sorry this is the instructions you need for doing security releases. Can you get this into the repo some time?

**Comment by unclecheese** [comment]
This is great. Would have been really nice to have this information for this last release.

**Pull request merged**